### ISSUE TITLE
Addon breaks GraphQL functionality

### ISSUE THREAD

**Issue by St3Ko** [WithCandour/statamic-aardvark-seo#133]
this addon currently breaks GraphQL functionality since Statamic 3.3

Every query, which includes an entry with the blueprint `OnPageSeoBlueprint.php` attached will fail with an error of `Type GridItem_LocalizedUrls not found`.

removing the lines where the field `localized_urls` gets defined removes the error. (OnPageSeoBlueprint.php:92)

for more infos see [here](https://github.com/statamic/cms/issues/5648)

thx!

Statamic 3.3.11 Pro
Laravel 8.83.13
PHP 7.4.28
aryehraber/statamic-uuid 2.1.0
withcandour/aardvark-seo 2.0.24

**Comment by AndrewHaine** [comment]
Thank  you for reporting this, we will take a look into whether this is a required change on our end or in Statamic.

**Comment by marcdap** [comment]
Hello, I'm writing to know, if it's possible, the status of this error. We are also using the plugin, and it is breaking the functionality when querying using GraphQL.

Statamic: 3.3.35
Aardvark SEO: 2.0.28

As an example this query is working when the add-on is uninstalled:

```
query MyQuery {
  entries {
    from
    to
    data {
      id
      title
      slug
      collection {
        handle
      }
    }
  }
}
```

With the add-on installed the following error is thrown:

```
Type GridItem_LocalizedUrls not found.\nCheck that the config array key for the type matches the name attribute in the type's class.\nIt is required when 'lazyload_types' is enabled
```

Any information would be appreciated. Thank you.

**Comment by tomgillett** [comment]
I am also experiencing this same issue.

**Comment by goellner** [comment]
@AndrewHaine when are planning on making a new release with the fix? I tried our project with this fix and everything is building properly and the graphql queries are working again.

**Comment by AndrewHaine** [comment]
Hi @goellner , that'll be out later today :)

**Comment by AndrewHaine** [comment]
Fixed in 2.0.29
